epub33/core/index.html

```diff
@@ -2873,128 +2932,87 @@ <h6>Examples</h6>
 </pre>
 								</aside>
 
-								<aside class="example" id="example-manifest-flbk">
-									<p>The following example shows a <code>manifest</code> that references two
-											<a>Foreign Resources</a> and therefore uses the <a
-											href="#sec-foreign-restrictions-manifest">fallback chain mechanism</a> to
-										supply content alternatives. The fallback chain terminates with a <a>Core Media
-											Type Resource</a>.</p>
-									<pre>&lt;manifest&gt;
-    &lt;item id="item1" 
-          href="chap1_docbook.xml" 
-          media-type="application/docbook+xml" 
-          fallback="fall1"/&gt;
-    &lt;item id="fall1" 
-          href="chap1.xml" 
-          media-type="application/z3998-auth+xml" 
-          fallback="fall2" /&gt;
-    &lt;item id="fall2" 
-          href="chap1.xhtml" 
-          media-type="application/xhtml+xml"/&gt; 
-    … 
+								<aside class="example" id="example-manifest-flbk"
+									title="Foreign Resource with Fallback in Spine">
+									<p>The following example shows the <a href="#sec-foreign-restrictions-manifest"
+											>fallback chain mechanism</a> allowing a <a>Foreign Resource</a> (JPEG) to
+										be listed in the spine with fallback to an SVG Content Document.</p>
+									<pre>Spine:
+&lt;itemref idref="page-001"/&gt;
```

Comment:
I am bothered (here and the other examples) by the style of the example. In all the examples, we provide an extract of a bona fide package XML file with, usually, the spine following the manifest part. In this example (and the other examples in the section) we separate these, and it took me a certain time to realize that. What I would have preferred is something like:

```xml
<package …>
    <manifest>
        …
        <item id="page-001" …>
        <item id="page-001-svg" …>
        …
    </manifest>
    <spine>
        …
        <itemref idref="page-001" />
        …
    </spine>
</package>